object_store/Cargo.toml

```diff
@@ -46,7 +46,7 @@ walkdir = "2"
 # Cloud storage support
 base64 = { version = "0.21", default-features = false, features = ["std"], optional = true }
 hyper = { version = "0.14", default-features = false, optional = true }
-quick-xml = { version = "0.30.0", features = ["serialize", "overlapped-lists"], optional = true }
+quick-xml = { git = "https://github.com/tafia/quick-xml.git", rev="db8546a", features = ["serialize", "overlapped-lists"], optional = true }
```

Comment:
in the latest version of the library (not released yet) escape level was decreased to `Partial`, meaning it would not escape `"` by default